src/components/elevation/Elevation.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentStory } from '@storybook/react'
+import Elevation from './Elevation'
+import PaddingBox from '@/components/padding-box/PaddingBox'
+import { Size } from '@/types/common'
+
+export default {
+  title: 'Components/Elevation',
+  component: Elevation,
+}
+
+const Template: ComponentStory<typeof Elevation> = (args) => (
+  <Elevation {...args}>
+    <PaddingBox x="md" y="md">
+      Elevated Contents
+    </PaddingBox>
+  </Elevation>
+)
+
+export const Basic = Template.bind({})
+Basic.args = { elevation: 'sm' }
```

Comment:
I'd personally rather have a dropdown than a text box to fill out.
```suggestion
Basic.args = { elevation: 'sm' }
Basic.argTypes: {
  elevation: {
    options: ['none', 'xxs', 'xs', 'sm', 'md', 'lg', 'xl', 'xxl'],
    control: { type: 'radio' },
  },
}
```